app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -111,7 +112,14 @@ class RootActivity : AppCompatActivity(), RootScreen {
         setContent {
             val viewModel: RootViewModel = viewModel()
             val isSystemInDarkTheme = isSystemInDarkTheme()
+            val isDarkThemeEnabled = when (appDesign(ivyContext).context().theme) {
```

Comment:
Don't copy paste this code. Extract this condition as a function and re-use it. Copy pasting the same logic bumps the complexity and opens the room for error

Reply:
`    private fun isDarkThemeEnabled(ivyDesign: IvyDesign, systemDarkTheme: Boolean): Boolean {
        return when (ivyDesign.context().theme) {
            Theme.LIGHT -> false
            Theme.DARK -> true
            Theme.AMOLED_DARK -> true
            else -> systemDarkTheme
        }
    }` like this?

Reply:
yes, something like this and re-use it from wherever it was copied. My idea is to have a single-source-of-truth for that logic.

Reply:
Thank you, I'll try again :)